### DISCUSSION TITLE
Where in the Data? a new datagame

### DISCUSSION THREAD

**Discussion by anjakefala** [saulpw/visidata#2344] [Show and tell]
Hi everyone!

Since the release of VisiData 3.0 at the start of the year, we have hinted that we have been working on a `$SecretProject`.

We're excited to announce that we're at work on another data game, "Damaged Goods". It is scheduled to be released on May 6th, 2024.  We're planning a release event that week, when you and all your data friends can play the game and solve the puzzles.  

[The full announcement is here. =)](https://www.whereinthedata.com/news/announcing-damaged-goods/)

We would love your help for ensuring that this is just as successful as our last game! From [Saul's patreon](https://www.patreon.com/posts/announcing-new-99753700). 

1. **Mark your calendar**!  Set aside some time to play in the week of May 6th.  We're aiming for it to take about 3-5 hours in total to solve all the puzzles.

2. **Post the announcement around social media**!  Spread the word in your tech community about the upcoming game, and send people to [whereinthedata.com](https://whereinthedata.com/) to sign up for the mailing list.

3. **We're looking for a few good sponsors**!  If your company does any kind of data stuff, they might be interested in sponsoring this game.  This is a great way to get their name in front of a few thousand data nerds, whether for recruiting or brand awareness.  Send [whereinthedata.com/sponsor](https://whereinthedata.com/sponsor) to someone in your marketing department and tell them how much you enjoyed playing Noah's Rug (previously, Hanukkah of Data) last year (if you did, of course).

4. Help us test out the new client!  You can play through [Noah's Rug](https://whereinthedata.com/noahsrug) using SQLite, and send any feedback you have to [bugs@whereinthedata.com](mailto:bugs@whereinthedata.com).

If you're interested in getting a few more updates, make sure you've signed up for the data-games-specific mailing list on [whereinthedata.com](http://whereinthedata.com/).

If you want to talk with us about the game, come by the [Discord #datagames](https://discord.gg/A3wH6jnn5x) channel and say hi!

**Comment by slinkeepy** [comment]
Genuinely curious, has this been cancelled? I really enjoyed Noah's Rug, and would love something similar!

Or could you recommend any similar story-based data puzzles from other creators?

**Reply by anjakefala** [reply]
@slinkeepy Thanks so much for your interest. =)

It is still in the works! We did a lot of production in a short period of time, but the beta test showed that we missed the mark a little. We didn't want to rush to publish the game. It went back into production, and we're working on it actively. With full-time jobs, it's going to take some time. 

I haven't played this one yet, so I can't actually recommend it, but it piqued my interest! https://store.steampowered.com/app/3950130/Database_Detective_Minor_Crimes_Division/

**Reply by slinkeepy** [reply]
That's great @anjakefala, thanks so much for the quick reply!
I support the "It's ready when it's ready" philosophy, so I'm just glad it's still in the works...
And thanks for the Steam recommendation, I'll take a look :)

Is it possible to get a small update on the WITD site? I know I was very confused when I went there. Even the day/month beta date has no accompanying year context, and it's unclear if the beta sign-up is still (or was ever) open.

**Comment by anjakefala** [comment]
@slinkeepy Good idea! I'll work on an update. =)
